### ISSUE TITLE
v1 vs v2 commands

### ISSUE THREAD

**Issue by Zalastax** [tidalcycles/tidal-chocolatey#10]
In https://github.com/tidalcycles/tidal-chocolatey/pull/9#issuecomment-1291657367, @yaxu writes

> Because the v2- commands are full of regressions, If you install via v2-install, generally that means that later upgrades don't work.. So I'd stick with v1- for as long as we can.

It is not clear to me what full of regressions refers to here. Is it a correct interpretation that you refer to cabal not handling global libraries well? But it worked somewhat better in v1 than in v2 commands? If so, is our best hope to escape Haskell issues by improving the Tidal binary support? Do you think we could describe our needs / the problem we are facing and open an issue in Cabal?

**Comment by yaxu** [comment]
Yes basically I meant cabal v2- is broken for people like us who want to install and use libraries, rather than build binaries. I've had some discussions on the cabal issue tracker and the answer there has been that we are not using cabal in the way they expect. The answer to this is the development of a '[cabal env](https://github.com/haskell/cabal/issues/6481)' tool but despite a working prototype this seems to have stalled.

Agreed a binary distribution of tidal would be a great way out of this.
